### PR TITLE
DOCS-12816: Wildcard Index Touch Ups

### PR DESCRIPTION
**Pull request by rkumar-mongo**

### CODE REVIEW

**Comment by kay-kim** [comment]
@rkumar-mongo   is that extract used anywhere else?

**Comment by rkumar-mongo** [comment]
Just the wildcard index page itself. I think the change actually benefits that page as well.

File: source/includes/extracts-wildcard-indexes.yaml [hunk 1]
```diff
@@ -45,15 +45,21 @@ content: |
      { "userMetadata" : "inactive" }
 
   Administrators want to create indexes to support queries on any
-  subfield of ``userMetadata``. A wildcard index on ``userMetadata``
+  subfield of ``userMetadata``.
```
Comment (kay-kim) [inline source/includes/extracts-wildcard-indexes.yaml]: Above -- line 42 is not valid

Comment (rkumar-mongo) [reply source/includes/extracts-wildcard-indexes.yaml]: Fixed

**Pull request merged**